src/main/java/step4/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,33 @@
+package step4.domain.lotto;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+/*
+ * Lotto
+ * ver. 1.0
+ * 2020.05.31
+ */
+public class Lotto {
+
+    // each Lotto numbers.
+    private Set<Integer> lottoNumbers;
+
+    public Lotto(TreeSet<Integer> lottoNumbers) {
```

Comment:
```suggestion
    public Lotto(Set<Integer> lottoNumbers) {
```
구체적인 클래스 선언 보다는 다형성을 활용하기 위한 상위 인터페이스를 파라미터로 받아주세요 😄

Reply:
네 , 적용하겟습니다.